pkg/virt-controller/services/renderresources.go

```diff
@@ -212,14 +212,20 @@ func WithCPUPinning(cpu *v1.CPU) ResourceRendererOption {
 			}
 		}
 
-		// allocate 1 more pcpu if IsolateEmulatorThread request
+		// allocate pcpus for emulatorThread if IsolateEmulatorThread is requested
 		if cpu.IsolateEmulatorThread {
-			emulatorThreadCPU := resource.NewQuantity(1, resource.BinarySI)
+			const emulatorThreadCPU = 1
+			emulatorThreadCPUs := resource.NewQuantity(emulatorThreadCPU, resource.BinarySI)
```

Comment:
AFAIK `resource.BinarySI` is used to define resources of type memory or storage.
I think it should be `resource.DecimalSI`.

Reply:
It's not directly related to this change, Can we pin this for a separate PR?

Reply:
> It's not directly related to this change. Can we pin this for a separate PR?

Reply:
Because the values are one and two, it doesn't seem to matter, but in the tests there are decimal numbers being used `1000m` etc so it is a bit confusing IMO.

Reply:
I agree, but it's a discussion that deserves a separate PR IMO.